ci-operator/step-registry/redhat-chaos/random-system-pods-disruption/redhat-chaos-random-system-pods-disruption-command.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python3 --version
+pushd /tmp
+
+ls -la /root/kraken
+git clone https://github.com/redhat-chaos/krkn-hub.git
+pushd krkn-hub/
+
+echo "kubeconfig loc $$KUBECONFIG"
+
+export KRKN_KUBE_CONFIG=$KUBECONFIG
+export NAMESPACE="openshift-.*"
```

Comment:
I think there's a way to export an enviornment variable at a higher level so we could keep the scenarios at "pod-scenarios" but at the workflow level of set the variables we want to run. This way if one thing changes we dont need to rewrite every file

Reply:
documentation: https://docs.ci.openshift.org/docs/architecture/ci-operator/#dependency-overrides